docs/source-app/workflows/ssh/index.rst

```diff
@@ -0,0 +1,83 @@
+
+#################################
+Access your Apps via SSH
+#################################
+
+**Audience:** Users who want to debug their Flow and Work in a the runtime environment.
+
+----
+
+**********************************************************
+Generating a new SSH key and configure ssh-agent
+**********************************************************
+Open a terminal and run the following command:
+
+.. code:: bash
+
+   $ ssh-keygen -t ed25519 -C "your_email@example.com"
+
+This creates a new SSH key, using the provided email as a label.
+
+At the prompt, type a secure passphrase.
+
+.. code:: bash
+
+   > Enter passphrase (empty for no passphrase): [Type a passphrase]
+   > Enter same passphrase again: [Type passphrase again]
+
+Next, start your ssh-agent in the background:
+
+.. code:: bash
+
+   $ eval "$(ssh-agent -s)"
+   > Agent pid 12345
+
+Add your generated ssh key:
+
+.. code:: bash
+
+   $ ssh-add ~/.ssh/id_ed25519
+
+Lastly, verify your ssh-key is properly loaded:
+
+.. code:: bash
+
+   $ ssh-add -L
+   ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAn8mYRnRG1banQcfXPCUC6R8FvQS+YgfIsl70/dD3Te your_email@example.com
+
+----
+
+**********************************************************
+Managing SSH keys with lightning CLI
+**********************************************************
+Open a terminal and run the following command:
+
+.. code:: bash
+
+   $ lightning add ssh-key --public-key ~/.ssh/id_ed25519.pub
+
+Next, list your configured SSH keys:
+
+.. code:: bash
+
+   $ lightning list ssh-keys
+
+You are now ready to access your Lightning Flow and Work containers.
+
+----
```

Comment:
```suggestion

Before you can SSH to cloud machines, you will need to generate a new private SSH key, add it to the SSH agent, and add the public SSH key to your account on Lightning.


Step 1: Create an SSH key
===================

Open a terminal and run the following command (replace email with the address you used in your lightning.ai account):

.. code:: bash

   # make the ssh key (if you don't have one)
   $ ssh-keygen -t ed25519 -C "your_email@example.com"

This creates a new SSH key, using the provided email as a label.

At the prompt, type a secure passphrase.

.. code:: bash

   > Enter passphrase (empty for no passphrase): [Type a passphrase]
   > Enter same passphrase again: [Type passphrase again]


Step 2: add the key to the ssh-agent
===========================

Next, start your ssh-agent in the background:

.. code:: bash

   # add the key to the ssh-agent (to avoid having to explicitly state key on each connection)
   # to start the agent, run the following
   $ eval "$(ssh-agent -s)"
   > Agent pid 12345

Add your generated ssh key:

.. code:: bash

   $ ssh-add ~/.ssh/id_ed25519

Lastly, verify your ssh-key is properly loaded:

.. code:: bash

   $ ssh-add -L
   ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAn8mYRnRG1banQcfXPCUC6R8FvQS+YgfIsl70/dD3Te your_email@example.com

Step 3: Add your key to Lightning
=========================

You can add SSH keys using Lightning.ai UI, or via this CLI command:

.. code:: bash

   $ lightning add ssh-key --public-key ~/.ssh/id_ed25519.pub

Next, list your configured SSH keys:

.. code:: bash

   $ lightning list ssh-keys

You are now ready to access your Lightning Flow and Work containers.

----
```